protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -1186,52 +1293,68 @@ message ListCandleDataResponse {
     CandleDataConnection candles = 1;
 }
 
+// Represents the candle data with the corresponding cursor for it
 message CandleEdge {
+    // List of candle data chunks
     Candle node = 1;
+    // Cursor that can be used to fetch futher pages
     string cursor = 2;
 }
 
+// Represents the list of Candle data objects with the list page info
```

Comment:
What is "the list page info"?

Reply:
it means the page info content for the list - it tells you on which page you are in the list of candles